test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -168,18 +168,39 @@ protected async Task WaitForRaftIndexToBeAppliedInCluster(long index, TimeSpan t
             if (await Task.WhenAll(tasks).WaitAsync(timeout))
                 return;
 
-            var message = $"Timed out after {timeout} waiting for index {index} because out of {Servers.Count} servers" +
+            ThrowTimeoutException(Servers, tasks, index, timeout);
+        }
+
+        protected async Task WaitForRaftIndexToBeAppliedOnClusterNodes(long index, List<RavenServer> nodes, TimeSpan? timeout = null)
```

Comment:
I think we should use this method in place of `WaitForRaftIndexToBeAppliedInCluster` in every cluster test that run in parallel (all tests but rachis?)